Refactor Logic & Bug Fixes

T4E-JAnderson opened the pull request:
Reformat logic
Add Regions
Fix issues with return data
Add useful date info (based on Terms)
Convert Tabs to Spaces

T4E-JAnderson committed:
Refactor Logic & Bug Fixes

Reformat logic
Add Regions
Fix issues with return data
Add useful date info (based on Terms)
Convert Tabs to Spaces

T4E-JAnderson committed:
Add OneRoster 1.2 support and bug fixes

Fix retry logic.
Remove Demographics data fetch (not used)
Add additional SkipFields based on OneRoster 1.2 Spec
Add additional logic to handle non 'active' records to give them a lower priority.

msheldont4e reviewed (dismissed):
This is the v1.1 connector not v1.2

T4E-JAnderson committed:
Remove 1.2 Mention

Removed mention of 1.2.  The 1.2 logic will be put into a new Repo once verified.

T4E-JAnderson committed:
Update configuration.json

Fix Version URI Label
